src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3422,100 +3328,86 @@ namespace HVACUnitaryBypassVAV {
+        if (CBVAV(CBVAVNum).plenumIndex > 0 || CBVAV(CBVAVNum).mixerIndex > 0) {
+            int plenumOrMixerInletNode = CBVAV(CBVAVNum).PlenumMixerInletAirNode;
+            DataLoopNode::Node(plenumOrMixerInletNode) = DataLoopNode::Node(splitterOutNode);
+            DataLoopNode::Node(plenumOrMixerInletNode).MassFlowRate =
+                BypassDuctFlowFraction * DataLoopNode::Node(CBVAV(CBVAVNum).MixerInletAirNode).MassFlowRate;
+            DataLoopNode::Node(plenumOrMixerInletNode).MassFlowRateMaxAvail = DataLoopNode::Node(plenumOrMixerInletNode).MassFlowRate;
+            DataAirLoop::AirLoopFlow(CBVAV(CBVAVNum).AirLoopNumber).BypassMassFlow = DataLoopNode::Node(plenumOrMixerInletNode).MassFlowRate;
+        }
 
         // calculate sensible load met using delta enthalpy at a constant (minimum) humidity ratio)
-        MinHumRat = min(Node(InletNode).HumRat, Node(OutletNode).HumRat);
-        LoadMet = Node(OutletNode).MassFlowRate * (PsyHFnTdbW(Node(OutletNode).Temp, MinHumRat) - PsyHFnTdbW(Node(InletNode).Temp, MinHumRat));
+        MinHumRat = min(DataLoopNode::Node(InletNode).HumRat, DataLoopNode::Node(OutletNode).HumRat);
+        LoadMet = DataLoopNode::Node(OutletNode).MassFlowRate * (Psychrometrics::PsyHFnTdbW(DataLoopNode::Node(OutletNode).Temp, MinHumRat) -
+                                                                 Psychrometrics::PsyHFnTdbW(DataLoopNode::Node(InletNode).Temp, MinHumRat));
     }
 
-    void GetZoneLoads(int const CBVAVNum, // Index to CBVAV unit being simulated
-                      Real64 &QZoneReq    // Total zone load served by this air loop
+    void GetZoneLoads(int const CBVAVNum // Index to CBVAV unit being simulated
     )
     {
 
         // SUBROUTINE INFORMATION:
         //       AUTHOR         Richard Raustad
         //       DATE WRITTEN   July 2006
-        //       MODIFIED       na
-        //       RE-ENGINEERED  na
 
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine is used to poll the thermostats in each zone and determine the
         // mode of operation, either cooling, heating, or none.
 
-        // METHODOLOGY EMPLOYED:
-        // na
-
-        // REFERENCES:
-        // na
-
-        // Using/Aliasing
-        using DataZoneEnergyDemands::CurDeadBandOrSetback;
-        using DataZoneEnergyDemands::ZoneSysEnergyDemand;
-
-        // Locals
-        // SUBROUTINE ARGUMENT DEFINITIONS:
-
-        // SUBROUTINE PARAMETER DEFINITIONS:
-        // na
+        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+        Real64 ZoneLoad = 0.0; // Total load in controlled zone [W]
 
-        // INTERFACE BLOCK SPECIFICATIONS
-        // na
+        Real64 thisTime = (DataGlobals::DayOfSim - 1) * 24 + DataGlobals::HourOfDay - 1 + (DataGlobals::TimeStep - 1) * DataGlobals::TimeStepZone +
+                          DataHVACGlobals::SysTimeElapsed;
 
-        // DERIVED TYPE DEFINITIONS
-        // na
+        if (thisTime <= CBVAV(CBVAVNum).changeOverTimer) return;
```

Comment:
Compare time of this time step to time where change over can be allowed, if no change over is allowed return.